pkgpanda/http/__init__.py

```diff
@@ -0,0 +1,101 @@
+"""Pkgpanda HTTP API"""
+
+import http.client
+import sys
+
+from flask import Flask, abort, current_app, json, request
+
+from pkgpanda import Install, Repository, actions
+
+empty_response = ('', http.client.NO_CONTENT)
+
+
+def package_index(packages, package_id=None):
+    if package_id is None:
+        return json.dumps(sorted(packages))
+    elif package_id in packages:
+        return empty_response
```

Comment:
Depends on what model you are going after. Some people see the HTTP status codes as a "container" for the internal messages. So that the service processed your request and can return properly gets you a 200, then we're explicit in the body about what it's about.

Others say we're going to promote all internal errors to whatever is the most useful external error.

In this case, I think it's good to be explicit with the extra message / data inside the body in addition to the "200" response. Doesn't break anything, makes it so that if we ever decide to expand / change the endpoint in the future we have a good upgrade path.